app/helpers/notifications_helper.rb

```diff
@@ -95,14 +102,27 @@ def no_url_filter_parameters_present
     notification_param_keys.all?{|param| params[param].blank? }
   end
 
-  def notification_icon(subject_type)
+  def notification_icon(subject_type, state = nil)
+    return 'issue-closed' if subject_type == 'Issue' && state == 'closed'
```

Comment:
I get why you did this, but feels a bit awkward now that we have more than one decision to make (unlike the other hashes). Maybe, since we only use that constant hash in this spot, we should just put all the logic in here? Like

```ruby
def notification_icon(subject_type, state = nil)
  case subject_type
  when 'Issue'
    if state == 'closed'
      'issue-closed'
    else
      'issue-open'
    end
  when 'PullRequest' then 'git-pull-request'
  when 'Commit' then 'git-commit'
  # ...etc...
  end
end
```